packages/xo-web/src/common/xo/index.js

```
@@ -1010,29 +1032,55 @@ export const startVms = vms =>
       )
     )
 
-    if (forbiddenStart.length === 0) {
+    if (forbiddenStart.length === 0 && vmsWithduplicatedMacAddresses.length === 0) {
       if (nErrors === 0) {
         return
       }
 
       return error(_('failedVmsErrorTitle'), _('failedVmsErrorMessage', { nVms: nErrors }))
     }
 
-    const choice = await chooseActionToUnblockForbiddenStartVm({
-      body: _('blockedStartVmsModalMessage', { nVms: forbiddenStart.length }),
-      title: _('forceStartVmModalTitle'),
-    }).catch(noop)
-
-    if (nErrors !== 0) {
-      error(_('failedVmsErrorTitle'), _('failedVmsErrorMessage', { nVms: nErrors }))
+    if (vmsWithduplicatedMacAddresses.length > 0) {
+      // Retry without checking MAC addresses
+      try {
+        await confirm({
+          title: _('forceStartVm'),
+          body: _('vmsWithDuplicatedMacAddressesMessage', { nVms: vmsWithduplicatedMacAddresses.length }),
+        })
+        await Promise.all(
+          map(vmsWithduplicatedMacAddresses, id =>
+            _startVm(id, undefined, { bypassMacAddressesCheck: true }).catch(reason => {
+              if (forbiddenOperation.is(reason)) {
+                forbiddenStart.push(id)
+              } else {
+                nErrors++
+              }
+            })
+          )
+        )
+      } catch (error) {
+        // Nothing to do
```

Let it throw: if the user didn't confirm, it has to stop.